sqrl-examples/avro/README.md

```diff
@@ -0,0 +1,68 @@
+# Avro for SQRL: Setup and Execution Guide
+
+## Compile SQRL Script
+Compile your .sqrl and schema.graphqls using Docker:
+```shell
+docker run --rm -v $PWD:/build datasqrl/cmd compile c360.sqrl schema.graphqls --mnt $PWD
+```
+
+## Add Kafka Topics
+Create a script to add orders and ordercount-1 topics in Kafka:
+```shell
+echo '
+#!/bin/bash
+/opt/bitnami/kafka/bin/kafka-topics.sh --create --bootstrap-server kafka:9092 --topic orders --partitions 1 --replication-factor 1
```

Comment:
Do you think it might be easier to say "add this one line to the create-topics.sh file"?

Reply:
imo its easy to give a copy-paste command, but ill add words to that effect

Reply:
I was hoping to use an external kafka instance but it became too cumbersome to setup